app/services/modsulator_client.rb

```diff
@@ -69,20 +69,15 @@ def call_remote_modsulator(url, content_type)
     end
     response_xml.body
   rescue Faraday::ResourceNotFound => e
-    delayed_log_url(e, url)
-    log.puts "argo.bulk_metadata.bulk_log_invalid_url #{e.message}"
+    log_request_error(e, url, 'argo.bulk_metadata.bulk_log_invalid_url')
   rescue Errno::ENOENT => e
-    delayed_log_url(e, url)
-    log.puts "argo.bulk_metadata.bulk_log_nonexistent_file #{e.message}"
+    log_request_error(e, url, 'argo.bulk_metadata.bulk_log_nonexistent_file')
   rescue Errno::EACCES => e
-    delayed_log_url(e, url)
-    log.puts "argo.bulk_metadata.bulk_log_invalid_permission #{e.message}"
+    log_request_error(e, url, 'argo.bulk_metadata.bulk_log_invalid_permission')
   rescue Faraday::Error => e
-    delayed_log_url(e, url)
-    log.puts "argo.bulk_metadata.bulk_log_internal_error #{e.message}"
+    log_request_error(e, url, 'argo.bulk_metadata.bulk_log_internal_error')
```

Comment:
diff coverage is failing because this is the only tested case out of these refactored error handling code paths.  these are all pretty similar calls, the tested usage works as it did before, and overall coverage seems to have gone up just a bit since a few lines of code were dropped.   but also, happy to add simple tests for the other error cases if people prefer.